PIL/TiffImagePlugin.py

```diff
@@ -959,8 +959,10 @@ def is_animated(self):
         return self._is_animated
 
     def seek(self, frame):
+        if not self._seek_check(frame):
+            return
         "Select a given frame as current image"
```

Comment:
This was formerly a doc string, now it's a random string comment in the code.

Reply:
Okay, I've updated the commits to move it back to the start of the method.